runtime/include/blueoil.hpp

```diff
@@ -23,9 +23,17 @@ extern "C" {
 
 
 namespace blueoil {
-struct Tensor {
+class Tensor {
+public:
   std::vector<float> data;
   std::vector<int> shape;
```

Comment:
Should really `data` and `shape` remain to be directly accessible from outside, after we made Tensor to be a class?

Reply:
I've done apply private to shape, data.
https://github.com/blue-oil/blueoil/pull/9/commits/34d72c45774eb357663cb5e4c2e32869e7d1e1ed